index.html

```diff
@@ -53,11 +53,11 @@
                 Portfolio
               </a>
               <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownPortfolio">
-                <a class="dropdown-item" href="portfolio-1-col.html">1 Column Portfolio</a>
-                <a class="dropdown-item" href="portfolio-2-col.html">2 Column Portfolio</a>
-                <a class="dropdown-item" href="portfolio-3-col.html">3 Column Portfolio</a>
-                <a class="dropdown-item" href="portfolio-4-col.html">4 Column Portfolio</a>
-                <a class="dropdown-item" href="portfolio-item.html">Single Portfolio Item</a>
+                <a class="dropdown-item" href="portfolio-1-col.html">Portfolio 1 Column</a>
+                <a class="dropdown-item" href="portfolio-2-col.html">Portfolio 2 Column</a>
```

Comment:
These seem like clearer names to give our users an idea of what these elements are on our page, rather than relying on what the template gives us.